CHANGELOG.md

```diff
@@ -51,6 +51,15 @@ Our versioning strategy is as follows:
 
 ### 🧹 Chores
 
+## 22.1.1
+
+### 🎉 New Features & Improvements
+
+NOTE: API was changed, preview data provides a new parameter `sc_layoutKind` therefore, please make the necessary updates to the nextjs template files as shown in the following PR to experience a smooth upgrade.
```

Comment:
Can we follow our format of CHANGELOG notes and write some required changes under the specific note? Under line 60 for example
Also this message is too generic at first glance, although it's specific to XM Cloud and Metadata mode.
```suggestion
* [XM Cloud] [Metadata Mode] [Next.js] API was changed, next.js preview data provides a new parameter `layoutKind` therefore, please make the necessary updates in the app in "plugins/preview-mode.ts" as shown in the following PR to experience a smooth upgrade.
```